<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>900</width>
     <height>900</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>36</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P3:</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="P1">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="P2">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="P3">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>680</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QTextEdit" name="p3xEdit"/>
   <widget class="QTextEdit" name="p3yEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>679</y>
     <width>59</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P3 X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P3 Y:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="P3_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Unicode MS</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>P4:</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>771</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QTextEdit" name="p4xEdit"/>
   <widget class="QTextEdit" name="p4yEdit"/>
  </widget>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>59</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P4 X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>P4 Y:</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>101</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_19">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>23:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>23:</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>31</width>
     <height>371</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>23:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>13:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>23:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
   <slots>
    <signal>xEnterSignal()</signal>
    <signal>yEnterSignal()</signal>
    <slot>xEnterSlot()</slot>
    <slot>yEnterSlot()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
